dotcom-rendering/src/components/ShareButton.importable.tsx

```diff
@@ -0,0 +1,223 @@
+import { css } from '@emotion/react';
+import { palette, until } from '@guardian/source-foundations';
+import type { Size } from '@guardian/source-react-components';
+import {
+	Button,
+	SvgCheckmark,
+	SvgShare,
+} from '@guardian/source-react-components';
+import { useEffect, useState } from 'react';
+import { palette as themePalette } from '../palette';
+import LinkIcon from '../static/icons/link-icon.svg';
+
+type Props = {
+	size?: Size | undefined;
+	pageId: string;
+	blockId?: string;
+	webTitle: string;
+	format: ArticleFormat;
+};
+
+const sharedButtonStyles = (sizeXSmall: boolean) => css`
```

Comment:
Non-blocking suggestion: I wonder if the styles would be easier to parse if the sizeXSmall styles were extracted into their own styles. This would also mean we would need these to be functions. We could then decide the styles based on the variable in the style prop